src/sql/query.rs

```diff
@@ -1721,17 +1721,31 @@ fn plan_function<'a>(
                 }
 
                 let source_timestamp =
-                    plan_expr(ecx, &sql_func.args[1], Some(ScalarType::Timestamp))?;
+                    plan_expr(ecx, &sql_func.args[1], Some(ScalarType::TimestampTz))?;
                 let typ = ecx.column_type(&source_timestamp);
-                if typ.scalar_type != ScalarType::Timestamp {
-                    bail!("date_trunc() is currently only implemented for TIMESTAMPs");
+                if typ.scalar_type != ScalarType::Timestamp
```

Comment:
This check is redundant with the check below, no?

Reply:
oof yup! meant to rip this out and forgot